From c768cf0cd139c5eba074671235818026bacc7e69 Mon Sep 17 00:00:00 2001
From: Shadi Ammouri <shadi@marvell.com>
Date: Wed, 13 Jan 2016 13:35:54 +0200
Subject: [PATCH 0679/1240] pcie: cp110: Add cfg space for pcie driver

	Add configuration space range for CP110 pcie driver.

Change-Id: I80f3a29afd0f1a2941e855cdfde54a2becbbd3d8
Signed-off-by: Shadi Ammouri <shadi@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/26762
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm/dts/cp110.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/dts/cp110.dtsi b/arch/arm/dts/cp110.dtsi
index eb848f9..8100d39 100644
--- a/arch/arm/dts/cp110.dtsi
+++ b/arch/arm/dts/cp110.dtsi
@@ -93,16 +93,19 @@ pcie-controller {
 	pcie@0,0 {
 		reg = <0x00600000 0x20000>;
 		mem = <0xf6000000 0x1000000>;
+		cfg = <0xf6f00000 0x100000>;
 		status = "disabled";
 	};
 	pcie@1,0 {
 		reg = <0x00620000 0x20000>;
 		mem = <0xf7000000 0x1000000>;
+		cfg = <0xf7f00000 0x100000>;
 		status = "disabled";
 	};
 	pcie@2,0 {
 		reg = <0x00640000 0x20000>;
 		mem = <0xf8000000 0x1000000>;
+		cfg = <0xf8f00000 0x100000>;
 		status = "disabled";
 	};
 };
-- 
1.9.1

